### PR TITLE
Account payment method management

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open your account page
- if you have a payment method click the new `Remove payment details` next to the card (if not try the instructions below to create one first)
- the method should remove
- the section should reload and you should now see instructions to create a new payment method
- open the form, fill it out and click submit
- the section should reload and you should now see your payment method

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7127/

File: jujugui/static/gui/src/app/components/account/payment-methods/card/card.js [hunk 1]
```diff
@@ -44,6 +55,47 @@ YUI.add('account-payment-method-card', function() {
...
+      this.xhrs.push(xhr);
+    },
+
+    /**
+      Generate the card actions.
+
+      @method _generateActions
+    */
+    _generateActions: function() {
+      if (!this.props.removePaymentMethod) {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/account/payment-methods/card/card.js]: This component gets reused in the deployment flow, in that case we don't allow the payment method to be removed so we don't need those methods (but maybe we should, or at least link to the account page).

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7132/

**Comment by hatched** [comment]
:shipit: to allow continued development down this path

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**